tests/plugins/test_tvp.py

```diff
@@ -1,7 +1,6 @@
 from streamlink.plugins.tvp import TVP
 from tests.plugins import PluginCanHandleUrl
 
-
```

Comment:
Don't remove this blank line. It's here deliberately.

- https://github.com/streamlink/streamlink/actions/runs/10132998278/job/28017929129?pr=6097#step:5:23

Reply:
I run the ruff check.